src/store/getters/state-getters.ts

```diff
@@ -742,10 +742,14 @@ export const hasRecDeliveryChanged = (state: StateIF): boolean => {
 export const havePeopleAndRolesChanged = (state: StateIF): boolean => {
   /** Normalizes fields that may be empty, null or undefined. */
   function normalize (op: OrgPersonIF): OrgPersonIF {
-    if (!op.deliveryAddress.deliveryInstructions) op.deliveryAddress.deliveryInstructions = ''
-    if (!op.deliveryAddress.streetAddressAdditional) op.deliveryAddress.streetAddressAdditional = ''
-    if (!op.mailingAddress.deliveryInstructions) op.mailingAddress.deliveryInstructions = ''
-    if (!op.mailingAddress.streetAddressAdditional) op.mailingAddress.streetAddressAdditional = ''
+    if (op.deliveryAddress) {
+      if (!op.deliveryAddress.deliveryInstructions) op.deliveryAddress.deliveryInstructions = ''
+      if (!op.deliveryAddress.streetAddressAdditional) op.deliveryAddress.streetAddressAdditional = ''
+    }
+    if (op.mailingAddress) {
+      if (!op.mailingAddress.deliveryInstructions) op.mailingAddress.deliveryInstructions = ''
+      if (!op.mailingAddress.streetAddressAdditional) op.mailingAddress.streetAddressAdditional = ''
+    }
```

Comment:
This looks good, but did you see this code and would it be a good place to init a blank address?

https://github.com/bcgov/business-edit-ui/blob/51716ef295600d0a7b7aa0fb9293dca0b9d595f2/src/views/Conversion.vue#L210

Reply:
that make sense. will update

Reply:
I'm not sure which is better (more concise for the current issue, clearer to understand)... leaving it with you. 😊

Reply:
its failing without this if conditions. so not changing it

Reply:
OK